src/Templates/ReportProblem.tmpl.php

```diff
@@ -0,0 +1,33 @@
+<?php if (!defined('LOADED_FROM_INDEX') || LOADED_FROM_INDEX != 'true') { die('Access denied.'); }?>
+
+<?php use RobinTheHood\ModifiedModuleLoaderClient\SendMail; ?>
+
+<?php 
+if (isset($_POST['send_mail'])) {
+    SendMail::sendFeedback();
+} ?>
```

Comment:
Can you put logical code (line 3-8) in the your new Methode invokeReportProblem () in IndexController. I know that the IndexController has to much code. In the future we can split up the IndexController in Multiple Controllers. 😊